src/main/java/io/vlingo/actors/Grid.java

```diff
@@ -141,6 +141,7 @@ public void nodeJoined(final Id newNode) {
             outbound.relocate(
                 newNode, nodeId, Definition.SerializationProxy.from(actor.definition()),
                 address, actor.provideRelocationSnapshot(), actor.pending());
+            // TODO catch QuorumLostException and retry (stack/schedule actor)
```

Comment:
@VaughnVernon this would be the solution of handling `QuorumLostException` for when relocation triggers as new nodes are added to the cluster. WDYT?